coala_quickstart/generation/Settings.py

```diff
@@ -92,11 +95,14 @@ def generate_settings(project_dir, project_files, ignore_globs, relevant_bears):
         [ext for lang in lang_files for ext in extset[lang]],
         relevant_bears[lang_map["all"]])
 
-    settings["default"]["ignore"] = generate_ignore_field(
-        project_dir,
-        lang_files.keys(),
-        extset,
-        ignore_globs)
+    if generate_ignore_field(project_dir, lang_files.keys(), extset,
```

Comment:
Why call the same function twice? Call it once, store the result and reuse it twice ;)